docs/src/terminology.md

```diff
@@ -260,7 +263,11 @@ A scarce, fungible member of a set of tokens.
 
 ## transaction
 
-One or more [instructions](terminology.md#instruction) signed by the [client](terminology.md#client) and executed atomically.
+One or more [instructions](terminology.md#instruction) signed by the [client](terminology.md#client) using one or more [keypairs](terminology.md#keypair) and executed atomically with only two possible outcomes of success or failure.
+
+## transaction id
+
+The first [signature](terminology.md#signature) in [transaction](terminology.md#transaction) which can be used to uniquely identify transaction across the complete [ledger](terminology.md#transaction), ranging from the newest [block](terminology.md#block) to the [genesis block](terminology.md#genesis-block).
```

Comment:
```suggestion
The first [signature](terminology.md#signature) in a [transaction](terminology.md#transaction), which can be used to uniquely identify the transaction across the complete [ledger](terminology.md#ledger).
```

Reply:
I personally don't think the clarification of what comprises the complete ledger is necessary, but if you feel strongly about it, feel free to put it back.

Reply:
nah, I'm already satisfied my added verbosity at `ledger`. :)